pybit/__init__.py

```diff
@@ -634,7 +634,7 @@ def replace_conditional_order(self, **kwargs):
         if 'symbol' in kwargs and 'USDT' in kwargs['symbol']:
             suffix = '/private/linear/stop-order/replace'
         else:
-            suffix = '/open-api/stop-order/replace'
+            suffix = '/v2/private/stop-order/replace'
```

Comment:
Doing my work for me! I was about to push an update with changes to the endpoints in a couple days as well as some other things, but I'll just have your pull request go through first instead. Thanks for the updates!

Reply:
no problem at all, happy to contribute and thanks for sharing your work! :)